aikau/package.json

```diff
@@ -1,29 +1,30 @@
 {
-   "name": "Aikau",
-   "version": "0.0.1",
-   "devDependencies": {
-      "forever": "0.15.1",
-      "grunt": "0.4.5",
-      "grunt-contrib-clean": "1.0.0",
-      "grunt-contrib-connect": "1.0.0",
-      "grunt-contrib-copy": "1.0.0",
-      "grunt-contrib-jshint": "1.0.0",
-      "grunt-contrib-watch": "0.6.1",
-      "grunt-folder-list": "1.1.0",
-      "grunt-http": "2.0.1",
-      "grunt-jsdoc": "1.1.0",
-      "grunt-notify": "0.4.5",
-      "grunt-prompt": "1.3.3",
-      "grunt-shell-spawn": "0.3.10",
-      "grunt-wait-server": "0.2.1",
-      "intern": "3.0.6",
-      "jsdoc": "3.4.0",
-      "jshint-stylish": "2.1.0",
-      "load-grunt-tasks": "3.4.1",
-      "node-coverage": "1.0.7",
-      "properties-reader": "0.0.15",
-      "safe-json-serialiser": "0.0.3",
-      "tcp-port-used": "0.1.2",
-      "time-grunt": "1.3.0"
-   }
-}
\ No newline at end of file
+  "name": "Aikau",
+  "version": "0.0.1",
+  "devDependencies": {
+    "forever": "0.15.1",
+    "grunt": "0.4.5",
+    "grunt-contrib-clean": "1.0.0",
+    "grunt-contrib-connect": "1.0.0",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-jshint": "1.0.0",
+    "grunt-contrib-watch": "0.6.1",
+    "grunt-folder-list": "1.1.0",
+    "grunt-http": "2.0.1",
+    "grunt-jsdoc": "1.1.0",
+    "grunt-notify": "0.4.5",
+    "grunt-prompt": "1.3.3",
+    "grunt-shell-spawn": "0.3.10",
+    "grunt-wait-server": "0.2.1",
+    "intern": "3.0.6",
+    "jsdoc": "3.4.0",
+    "jshint-stylish": "2.1.0",
+    "less": "^2.6.1",
+    "load-grunt-tasks": "3.4.1",
+    "node-coverage": "1.0.7",
+    "properties-reader": "0.0.15",
+    "safe-json-serialiser": "0.0.3",
+    "tcp-port-used": "0.1.2",
+    "time-grunt": "1.3.0"
+  }
+}
```

Comment:
Not keen on the total whitespace reformat here that NPM does automatically. Will put it back to 3-spaces next time I update the dependencies, so no change needed at the moment.